packages/core/CHANGELOG.md

```diff
@@ -5,6 +5,35 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.11](https://github.com/waku-org/js-waku/compare/core-v0.0.10...core-v0.0.11) (2023-02-11)
+
+
+### ⚠ BREAKING CHANGES
+
+* bump typescript
+* bump all prod dependencies
+* bump libp2p dependencies
+
+### Features
+
+* Codec as a property of the protocol implementations ([a5ff788](https://github.com/waku-org/js-waku/commit/a5ff788eed419556e11319f22ca9e3109c81df92))
```

Comment:
This is actually quite an odd change? Do we know why it's included in the changelog? Is there a pacakge we haven't released since then? Fine to fix the changelog after the release in case it's just an issue because we run release please for the first time.

Reply:
This is because of how I set initial commit from which we want to make a release. 
https://github.com/waku-org/js-waku/blob/87ad83b252a913e6428135ec19cfd53b2c6c3a91/release-please-config.json#L2

But this change is the next commit:
![image](https://user-images.githubusercontent.com/118575614/218578910-ab8313ae-0935-4973-9d64-4fb1a9254d12.png)

So this is the reason it is included.

NB: bootstrap commit will be used only first time